DirectEmployers/MyJobs
# Fix radius and screenshot notifications for people who need them fixed

## What's this PR do?
Passes radius to the front end so that if the value is provided it can be persistent. 
Adds a path to the webhook. 

## How should this be manually tested?
(1) Ensure radius getting passed to the front end (on sites that radius is a valid term - see https://github.com/DirectEmployers/MyJobs/pull/3641 for instructions on how to make radius a valid parameter for a site. 
(2) I'm honestly not sure how to manually test the screenshot notification webhook. I'm open to questions that might lead to a meaningful discussion about how to test it.

## Any background context you want to provide?
The parties that have requested the changes are both prepared for their change to be in place (mostly in respect to the new webhook parameter).



## PR Checklist

### Separation of Concerns
